src/client/datascience/interactive-window/interactiveWindow.ts

```diff
@@ -115,7 +118,9 @@ export class InteractiveWindow extends InteractiveBase implements IInteractiveWi
         @inject(KernelSwitcher) switcher: KernelSwitcher,
         @inject(INotebookProvider) notebookProvider: INotebookProvider,
         @inject(UseCustomEditorApi) useCustomEditorApi: boolean,
-        @inject(IExperimentService) expService: IExperimentService
+        @inject(IExperimentService) expService: IExperimentService,
+        @inject(INotebookStorage) private notebookStorage: INotebookStorage,
+        @inject(IExportUtil) private exportUtil: IExportUtil
```

Comment:
```suggestion
        @inject(ExportUtil) private exportUtil: ExportUtil
```

I'd consider dumping the interface here.